src/imgui_ex/vcFileDialog.h

```diff
@@ -42,21 +43,12 @@ struct vcFileDialog
   vcFileDialogCallback onSelect;
 };
 
-void vcFileDialog_Show(vcFileDialog *pDialog, char *pPath, size_t pathLen, const char **ppExtensions, size_t numExtensions, vcFileDialogType dialogType, vcFileDialogCallback callback);
-
-template <size_t N> inline void vcFileDialog_Show(vcFileDialog *pDialog, char(&path)[N], const char **ppExtensions, size_t numExtensions, vcFileDialogType dialogType, vcFileDialogCallback callback)
```

Comment:
Why delete these two templates? It seems reasonable to me to have these two options?

Reply:
I'm trying to remove the function entirely in favour of using the vcIMSW file picker but the changes required to do that were beyond this PR